bigbluebutton-html5/imports/ui/components/breakout-room/breakout-room/running-room/component.tsx

```
@@ -0,0 +1,604 @@
+    if (breakouts.length === 0) return;
+
+    const raw = sessionStorage.getItem('breakoutInitialAssignments');
+    if (!raw) return;
+
+    try {
+      const assignments: Record<string, { roomSequence: number; userName: string }> = JSON.parse(raw);
+      sessionStorage.removeItem('breakoutInitialAssignments');
+      initialAssignmentsLoaded.current = true;
+
+      const seqToMeetingId = new Map<number, string>();
+      breakouts.forEach((b) => {
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)